libs/ddhub-client-gateway-frontend/ui/messaging/src/lib/containers/modals/MessageInboxDetails/MessageInboxDetails.tsx

```diff
@@ -169,7 +170,11 @@ export const MessageInboxDetails: FC = () => {
                     <MessageDetail
                       field={{
                         label: 'Message ID',
-                        value: inboxDetails.messageId,
+                        value:
+                          inboxDetails.messageId.length > 24
```

Comment:
I'm confused about this, what are you doing here?

Reply:
Comment added in the code